### ISSUE TITLE
Parallelizing execution of regressions in regtest.py 

### ISSUE THREAD

**Issue by zvonimir** [smackers/smack#83]
It would be good to add number of threads as a command line argument, and then execute regressions using the selected number of threads in parallel.

**Comment by jon-whit** [comment]
I'd be willing to tackle this one.

**Comment by jon-whit** [comment]
Did we want to merge the parallel tests? https://github.com/smackers/smack/pull/85

If there are no further suggestions/improvements that should be made at the moment, this would be a nice feature to have to speed up test execution.

**Comment by michael-emmi** [comment]
This has been merged into develop.
